mottaker/src/main/kotlin/no/nav/soknad/arkivering/soknadsmottaker/config/ApplicationConfig.kt

```diff
@@ -17,8 +18,18 @@ private val defaultProperties = ConfigurationMap(
 		"KAFKA_SECPROT" to "",
 		"KAFKA_SASLMEC" to "",
 
+		"KAFKA_SCHEMA_REGISTRY_USER" to "",
+		"KAFKA_SCHEMA_REGISTRY_PASSWORD" to "",
+		"KAFKA_SCHEMA_REGISTRY" to "",
+		"KAFKA_BROKERS" to "",
```

Comment:
Er denne ikke det samme som `KAFKA_BOOTSTRAP_SERVERS`? Kan vi finne tydligere navn som beskriver hva skillnaden er?

Reply:
Når vi i nais.yml til applikasjonen legger til kafka.pool, legges det automatisk inn ett sett properties i applikasjonens environment, inkludert KAFKA_BROKERS. Se https://doc.nais.io/persistence/kafka/application/ for oversikt over disse.

Reply:
Det visste jeg ikke. Magiskt!